### DISCUSSION TITLE
ForwardMessage套娃里的转发消息无法正常显示及打开

### DISCUSSION THREAD

**Discussion by Deficuet** [mamoe/mirai#1129] [Q&A] [answered]
捏了一个转发消息，里面塞了三条消息，一张图和另外两个转发消息，三个转发消息给的Contact是同一个群
现在是显示“查看3条转发消息”，console里显示的node也都齐全，但点开看后只有那张图，另外两条转发消息消失了。是Contact给错了吗

**Comment by Him188** [comment]
是bot发送还是接收

**Reply by Deficuet** [reply]
bot发送

**Comment by Him188** [comment, marked as answer]
跟踪 #1130

**Reply by Deficuet** [reply]
得嘞，辛苦了
